2024-11-20 15:54:56,046 INFO: Microblog startup [in D:\PROJECTS\FlaskHabr\app\__init__.py:47]
2024-11-20 15:55:17,220 ERROR: Exception on /user/john [GET] [in D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py:838]
Traceback (most recent call last):
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\app\routes.py", line 42, in user
    return render_template('user.html', user=user, posts=posts)
                                                         ^^^^^
NameError: name 'posts' is not defined. Did you mean: 'Post'?
2024-11-20 15:55:38,458 ERROR: Exception on /user/john [GET] [in D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py:838]
Traceback (most recent call last):
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\app\routes.py", line 42, in user
    return render_template('user.html', user=user, posts=posts)
                                                         ^^^^^
NameError: name 'posts' is not defined. Did you mean: 'Post'?
2024-11-20 16:01:55,507 INFO: Microblog startup [in D:\PROJECTS\FlaskHabr\app\__init__.py:47]
2024-11-20 16:01:59,250 ERROR: Exception on /user/john [GET] [in D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py:838]
Traceback (most recent call last):
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\app\routes.py", line 42, in user
    return render_template('user.html', user=user, posts=posts)
                                                         ^^^^^
NameError: name 'posts' is not defined. Did you mean: 'Post'?
2024-11-20 16:04:17,258 INFO: Microblog startup [in D:\PROJECTS\FlaskHabr\app\__init__.py:47]
2024-11-20 16:04:20,622 ERROR: Exception on /user/john [GET] [in D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py:838]
Traceback (most recent call last):
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\app\routes.py", line 42, in user
    return render_template('user.html', user=user, posts=posts)
                                                         ^^^^^
NameError: name 'posts' is not defined. Did you mean: 'Post'?
2024-11-20 16:18:54,353 INFO: Microblog startup [in D:\PROJECTS\FlaskHabr\app\__init__.py:47]
2024-11-20 16:18:57,523 ERROR: Exception on /user/john [GET] [in D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py:838]
Traceback (most recent call last):
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\app\routes.py", line 42, in user
    return render_template('user.html', user=user, posts=posts)
                                                         ^^^^^
NameError: name 'posts' is not defined. Did you mean: 'Post'?
2024-11-20 16:22:50,755 ERROR: Exception on /user/john [GET] [in D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py:838]
Traceback (most recent call last):
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\env\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PROJECTS\FlaskHabr\app\routes.py", line 42, in user
    return render_template('user.html', user=user, posts=posts)
                                                         ^^^^^
NameError: name 'posts' is not defined. Did you mean: 'Post'?
2024-11-20 16:25:38,244 INFO: Microblog startup [in D:\PROJECTS\FlaskHabr\app\__init__.py:47]
